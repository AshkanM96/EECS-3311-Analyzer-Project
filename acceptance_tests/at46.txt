open_set_enumeration
open_set_enumeration
integer_constant (1)
boolean_constant (false)
close_set_enumeration
open_set_enumeration
boolean_constant (true)
integer_constant (2)
close_set_enumeration
close_set_enumeration
evaluate
reset
reset
open_set_enumeration
open_set_enumeration
boolean_constant (true)
add
integer_constant (1)
integer_constant (1)
close_set_enumeration
open_set_enumeration
boolean_constant (false)
boolean_constant (false)
boolean_constant (true)
close_set_enumeration
close_set_enumeration
evaluate
reset
open_set_enumeration
open_set_enumeration
boolean_constant (true)
integer_constant (10)
close_set_enumeration
open_set_enumeration
boolean_constant (false)
boolean_constant (false)
boolean_constant (true)
boolean_constant (true)
boolean_constant (true)
boolean_constant (false)
boolean_constant (false)
close_set_enumeration
close_set_enumeration
evaluate
reset
open_set_enumeration
open_set_enumeration
integer_constant (2)
boolean_constant (true)
close_set_enumeration
open_set_enumeration
close_set_enumeration
close_set_enumeration
evaluate
type_check
evaluate
boolean_constant (true)
add
integer_constant (3)
integer_constant (9)
close_set_enumeration
close_set_enumeration
evaluate
type_check
