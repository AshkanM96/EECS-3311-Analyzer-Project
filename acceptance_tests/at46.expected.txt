  Expression currently specified: ?
  Report: Expression is initialized.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{?}}
  Report: OK.
->integer_constant(1)
  Expression currently specified: {{1, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{1, False, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, ?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{1, False}, {?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{1, False}, {True, ?}}
  Report: OK.
->integer_constant(2)
  Expression currently specified: {{1, False}, {True, 2, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, {True, 2}, ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, {True, 2}}
  Report: OK.
->evaluate
  Expression currently specified: {{1, False}, {True, 2}}
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, ?}}
  Report: OK.
->add
  Expression currently specified: {{True, (? + nil)}}
  Report: OK.
->integer_constant(1)
  Expression currently specified: {{True, (1 + ?)}}
  Report: OK.
->integer_constant(1)
  Expression currently specified: {{True, (1 + 1), ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, (1 + 1)}, ?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{True, (1 + 1)}, {?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, (1 + 1)}, {False, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, (1 + 1)}, {False, False, ?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, (1 + 1)}, {False, False, True, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, (1 + 1)}, {False, False, True}, ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, (1 + 1)}, {False, False, True}}
  Report: OK.
->evaluate
  Expression currently specified: {{True, (1 + 1)}, {False, False, True}}
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, ?}}
  Report: OK.
->integer_constant(10)
  Expression currently specified: {{True, 10, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, 10}, ?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{True, 10}, {?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, 10}, {False, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, 10}, {False, False, ?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, 10}, {False, False, True, ?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, 10}, {False, False, True, True, ?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{True, 10}, {False, False, True, True, True, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, 10}, {False, False, True, True, True, False, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{True, 10}, {False, False, True, True, True, False, False, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, 10}, {False, False, True, True, True, False, False}, ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{True, 10}, {False, False, True, True, True, False, False}}
  Report: OK.
->evaluate
  Expression currently specified: {{True, 10}, {False, False, True, True, True, False, False}}
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{?}}
  Report: OK.
->integer_constant(2)
  Expression currently specified: {{2, ?}}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {{2, True, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{2, True}, ?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{2, True}, {?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{2, True}, {?}}
  Report: Error: (Set enumeration must be non-empty).
->close_set_enumeration
  Expression currently specified: {{2, True}, {?}}
  Report: Error: (Set enumeration must be non-empty).
->evaluate
  Expression currently specified: {{2, True}, {?}}
  Report: Error (Expression is not yet fully specified).
->type_check
  Expression currently specified: {{2, True}, {?}}
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: {{2, True}, {?}}
  Report: Error (Expression is not yet fully specified).
->boolean_constant(True)
  Expression currently specified: {{2, True}, {True, ?}}
  Report: OK.
->add
  Expression currently specified: {{2, True}, {True, (? + nil)}}
  Report: OK.
->integer_constant(3)
  Expression currently specified: {{2, True}, {True, (3 + ?)}}
  Report: OK.
->integer_constant(9)
  Expression currently specified: {{2, True}, {True, (3 + 9), ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{2, True}, {True, (3 + 9)}, ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{2, True}, {True, (3 + 9)}}
  Report: OK.
->evaluate
  Expression currently specified: {{2, True}, {True, (3 + 9)}}
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: {{2, True}, {True, (3 + 9)}}
  Report: {{2, True}, {True, (3 + 9)}} is not type-correct.
